scripts/secondo-cli.sh

```
@@ -0,0 +1,350 @@
+  echo ""
+
+  # Display primary opinion
+  echo -e "${GREEN}━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━${NC}"
+  echo -e "${GREEN}🎯 Primary Opinion ($(echo "$parsed" | jq -r '.primary.modelDisplayName'))${NC}"
+  echo -e "${GREEN}━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━${NC}"
+  echo ""
+  echo "$parsed" | jq -r '.primary.response' | fold -s -w 80
+  echo ""
+
+  # Display secondary opinions
+  local secondary_count=$(echo "$parsed" | jq '.secondaryOpinions | length')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)